tests/test_actions.py

```diff
@@ -852,31 +835,6 @@ def test_implements_protocol(self) -> None:
         assert isinstance(q2, Describable)
         assert isinstance(q3, Describable)
 
-    def test_not_performable(self) -> None:
-        with pytest.raises(NotPerformable) as exc:
-            SilentlyPerformable(None)  # type: ignore
-
-        assert str(exc.value) == (
-            "SilentlyPerformable only works with Performables. "
-            "Use `Silently` instead."
-        )
-
-    def test_not_answerable(self) -> None:
-        with pytest.raises(NotAnswerable) as exc:
-            SilentlyAnswerable(None)  # type: ignore
-
-        assert str(exc.value) == (
-            "SilentlyAnswerable only works with Answerables. Use `Silently` instead."
-        )
-
-    def test_not_resolvable(self) -> None:
-        with pytest.raises(NotResolvable) as exc:
-            SilentlyResolvable(None)  # type: ignore
-
-        assert str(exc.value) == (
-            "SilentlyResolvable only works with Resolvables. Use `Silently` instead."
-        )
-
```

Comment:
I agree with what you said, it is cleaner 'cause now we don't have these bonus classes that someone might misuse accidentally!